plugins/tech-insights-backend/src/service/router.test.ts

```diff
@@ -44,6 +45,18 @@ describe('Tech Insights router tests', () => {
   });
 
   beforeAll(async () => {
+    const databaseManager: Partial<DatabaseManager> = {
+      forPlugin: () => ({
```

Comment:
could even breakout this inner part as a variable too and pass that into buildTechInsightsContext

Reply:
Hmm, sorry I'm not sure I follow, would you mind expanding?

Reply:
Ah it was just a suggestion - if you look a couple lines farther down, these inner parts appear as the argument to `buildTechInsightsContext`. Some duplication but it's not important at all, feel free to disregard

Reply:
It was a good one! I just wasn't thinking clearly. Resolving since it's cleaned up :D